mne/viz/evoked.py

```diff
@@ -181,7 +181,8 @@ def _plot_evoked(evoked, picks, exclude, unit, show, ylim, proj, xlim, hline,
                  units, scalings, titles, axes, plot_type, cmap=None,
                  gfp=False, window_title=None, spatial_colors=False,
                  set_tight_layout=True, selectable=True, zorder='unsorted',
-                 noise_cov=None):
+                 noise_cov=None, colorbar=True, mask=None, mask_style=None,
+                 mask_cmap=None, mask_alpha=.1):
```

Comment:
`mask_alpha=.3` everywhere else. Although I'd really like to see a lower default value (e.g. `0.2`), at least use consistent defaults.